modules/ROOT/pages/6.4.1-release-notes.adoc

```diff
@@ -617,11 +617,9 @@ As a result, the `onSetup` function for custom group toolbar buttons now runs co
 === An element could be dropped onto the descendants of an element with a `contenteditable="false"` attribute
 //#TINY-9364
 
-In {productname} 6.0, the drag and drop `config` did not check some conditions surrounding element placements using either a drag or drop action.  This affected users running {productname} 6.0, which would effectively allow descendants of `contenteditable="false"` element to be dropped into another `contenteditable="false"` element.
+In versions of {productname} since version 6.0, elements were allowed to be dropped onto descendants of `contenteditable="false"` elements.
```

Comment:
```suggestion
In {productname} 6.0 and later, elements could be dropped onto descendants of an element with a `contenteditable=false` attribute.
```
Suggested edit for consistency (‘X.y and later’ is our standard form of words) and correctness (`contenteditable` is an attribute an element can take, not a descriptor of what an element is).